outputs.tf

```diff
@@ -55,11 +55,11 @@ output "worker_security_group_id" {
 }
 
 output "worker_iam_role_name" {
-  description = "IAM role name attached to EKS workers"
+  description = "default IAM role name for EKS worker groups"
   value       = "${aws_iam_role.workers.name}"
 }
 
 output "worker_iam_role_arn" {
-  description = "IAM role ID attached to EKS workers"
+  description = "default IAM role name for EKS worker groups"
```

Comment:
Shouldn't this be "..role ARN..." too?

Reply:
Will fix - this is what generates the description in the README